Development/UID2SDKDevelopmentApp/UID2SDKDevelopmentApp/RootView.swift

```diff
@@ -14,36 +14,59 @@ struct RootView: View {
     private var viewModel = RootViewModel()
 
     @State
-    private var emailTextField = ""
-    
+    private var email = ""
+
+    @State
+    private var phone = ""
+
+    @State
+    private var isClientSide = true
+
     var body: some View {
         
         VStack {            
-            Text(viewModel.titleText)
+            Text("root.navigation.title")
                 .font(Font.system(size: 28, weight: .bold))
-                .padding(EdgeInsets(top: 0, leading: 16, bottom: 0, trailing: 16))
-            TextField("Email Address", text: $emailTextField)
-                .textFieldStyle(.roundedBorder)
-                .textCase(.lowercase)
-                .textContentType(.emailAddress)
-                .padding(EdgeInsets(top: 0, leading: 16, bottom: 0, trailing: 16))
-                .onSubmit {
-                    viewModel.handleEmailEntry(emailTextField.lowercased())
+            HStack {
+                TextField("Email Address", text: $email)
+                    .keyboardType(.emailAddress)
+                    .textInputAutocapitalization(.never)
+                    .textContentType(.emailAddress)
+                Button("Submit Email", systemImage: "arrow.right.circle.fill") {
+                    viewModel.handleEmailEntry(email, clientSide: isClientSide)
                 }
+                .labelStyle(.iconOnly)
+            }
+            HStack {
+                TextField("Phone", text: $phone)
+                    .keyboardType(.phonePad)
+                    .textContentType(.telephoneNumber)
+                Button("Submit Phone", systemImage: "arrow.right.circle.fill") {
+                    viewModel.handlePhoneEntry(phone, clientSide: isClientSide)
+                }
+                .labelStyle(.iconOnly)
+            }
+            Toggle(isOn: $isClientSide) {
+                Label("Client Side", systemImage: isClientSide ? "circle.fill" : "circle.slash")
+            }
+            .toggleStyle(.button)
```

Comment:
Nit: Looks like the selected vs unselected state has different heights. Might be worth fixing to avoid the vertical shift.